index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>review-own-code</title>
+</head>
+
+<body>
+    <div id="main">
```

Comment:
herre you must use semantic html tag (e.g section tag)

Reply:
- [ ] Good job so far for using the inline comments :+1:  But please phrase the comments politely, as suggestions instead of orders for example instead of "Here you must ..." you could have said, "Please use semantic html tag ..."  :smile: 

- [ ] Another thing, I would like to recommend you to always tell the reason, why you suggest something or request a change. An example will be:  Please use a semantic html tag .... because the <main> tag specifies the main content of a document. Please fix it :smile:

Reply:
@codecaiine thank you for your suggestion

Reply:
I would suggest you usage of semantic tags. for example using main tag right after body tag and then using section tags for each section inside main tag. its because usage of semantic tags helps in SEO and also it would portray as clean code.